.github/workflows/test.yaml

```
@@ -46,9 +46,52 @@ jobs:
     - name: List packages in environment
       run: pip list
     - name: Run tests
+      if: ${{ matrix.python-version == '3.6' }}
       shell: bash
       run: python -I -bb -X dev -W error -m pytest
+    - name: Run tests
+      if: ${{ matrix.python-version != '3.6' }}
+      shell: bash
+      run: python -I -bb -X dev -W error -m pytest --asyncio-mode=auto
     - name: Twine check
       run: pipx run twine check --strict dist/*
     - name: Pip check
       run: pip check -v
+
+  visual-regression-tests:
```

```suggestion
  visual-regression-test:
```

Since aside from trigger conditions this doesn't share anything with the other workflow, I suggest just sticking this in a separate workflow file so it can be rerun separately from the regular tests as needed (which is a significant pain point with GitHub actions). Also, we should be consistent about naming; either call this `visual-regression-test`, or rename the other job `test` (and same with the extras names).